entries/entry01.md

```diff
@@ -1,7 +1,55 @@
 # Entry 1
```

Comment:
Missing sources, EDP, skills. And I'm not taking off points for professionalism, but next time use more markdown syntax to create visual separation.